backend/src/apiserver/server/test_util.go

```diff
@@ -195,20 +178,6 @@ func initWithExperimentAndPipelineVersion(t *testing.T) (*resource.FakeClientMan
 	// Create a pipeline and then a pipeline version.
 	_, err = resourceManager.CreatePipeline("pipeline", "", "", []byte(testWorkflow.ToStringForStore()))
 	assert.Nil(t, err)
-	_, err = resourceManager.CreatePipelineVersion(&api.PipelineVersion{
```

Comment:
Why do we remove this part?

Reply:
Will convert to drag, will have to check some things and come back :)

Reply:
The reason I dropped this temporarily is that is has been breaking continuously(potentially since introduced) since we use the same UUID and get an error that we have previously not been checked. So the question is then does it really matter if we create this pipeline version or not @capri-xiyue? Same for the both 'initWithExperimentsAndTwoPipelineVersions' and 'initWithExperimentsAndPipelineVersions'